python/tests_no_import_change/test_no_import_change.py

```diff
@@ -0,0 +1,145 @@
+#
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# to run on cpu -> python test_no_import_change.py 0.2
+# to run on gpu -> python -m spark_rapids_ml test_no_import_change.py 0.2
+#
+# spark-submit based
+#
+# to run on cpu -> spark-submit --master local[1] test_no_import_change.py 0.2
+# to run on gpu -> spark-rapids-submit --master local[1] test_no_import_change.py 0.2
+#
+# notice no imports from spark_rapids_ml (except for verifying model types)
+#
+
+import sys
+import tempfile
+
+from pyspark.ml.classification import LogisticRegression, LogisticRegressionModel
+from pyspark.ml.functions import array_to_vector
+from pyspark.ml.tuning import CrossValidator
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col
+
+reg_param = float(sys.argv[1])
+
+data = [
+    ([1.0, 2.0], 1.0),
+    ([1.0, 3.0], 1.0),
+    ([2.0, 1.0], 0.0),
+    ([3.0, 1.0], 0.0),
+]
+schema = "features array<float>, label float"
+
+spark = SparkSession.builder.getOrCreate()
+df = spark.createDataFrame(data, schema=schema)
+df = df.select(array_to_vector(col("features")).alias("features"), "label")
+df.show()
+
+lr_estimator = LogisticRegression()
+print(lr_estimator.setFeaturesCol("features"))
+
```

Comment:
I added `lr_estimator.setStandardization(False)` but it ran with error

`AttributeError: 'LogisticRegression' object has no attribute 'setStandardization'. Did you mean: 'getStandardization'?`

No idea why.

Reply:
Turns out we never defined setStandardization for spark_rapids_ml LogisticRegression. adding to this PR.

Reply:
Yeah, In order to make no code change, seems we need to make sure that spark-rapids-ml contains the APIs of spark.ml

Reply:
but it could be in a follow up.

Reply:
Yes.  Current goal was to eliminate package import change.   We have quite a few gaps to address for full api compatibility.